tests/containers/bci_prepare.pm

```diff
@@ -106,6 +106,14 @@ sub run {
         record_info('IMAGE', $image);
         script_retry("$engine pull $image", timeout => 300, delay => 60, retry => 3);
         record_info('Inspect', script_output("$engine inspect $image"));
+
+        my $build = get_var('CONTAINER_IMAGE_BUILD');
+        if ($build && $build ne 'UNKNOWN') {
+            record_info('CONTAINER_IMAGE_BUILD', $build);
```

Comment:
Maybe this info is superfluous, we can remove it, it's already in the variable. 
Don't like much how this text fits in the frame:
![image](https://user-images.githubusercontent.com/11728254/173809495-b14c86e8-aaa2-46f1-94a1-2df3815a93d7.png)

Reply:
I did:
```suggestion
            record_info('BUILD#', "CONTAINER_IMAGE_BUILD=$build");
```